templates/CRM/common/standalone.tpl

```diff
@@ -25,11 +25,7 @@
       </ol></nav>
     {/if}
 
-    {if $standaloneErrors}
-      <div class="standalone-errors">
-        <ul>{$standaloneErrors}</ul>
-      </div>
-    {/if}
+    <div id="standalone-errors-wrapper"></div>
```

Comment:
Instead of adding this markup unconditionally, we could add it with the below jQuery only if errors exist.